src/main/java/mesosphere/marathon/metrics/Metrics.scala

```diff
@@ -64,6 +68,11 @@ class Metrics @Inject() (val registry: MetricRegistry) {
 }
 
 object Metrics {
+  class Counter(counter: com.codahale.metrics.Counter) {
```

Comment:
Counters and Gauges are a separate thing in codahale metrics. Don't know what to document here if it is statsd specific?